mcl_fieldpts_var: 0.3
mgauss_x: 2
mcl_pt2line_var: 60
mgauss_w: 2
mcl_heading_var: 0.323
mcl_ptangle_var: 0.4
vgauss_y: 3
mgauss_y: 3
mcl_aslow: 0.01
useAdaptiveParticle: false
mcl_afast: 0.1
vgauss_x: 5
mcl_line2line_var: 0.2